source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -176,6 +176,59 @@
 	line-height: var(--wp--custom--heading--level-5--typography--line-height);
 }
 
+.is-style-list-of-links {
+	--wporg--style--list-of-links--offset: 14px;
+	--wporg--style--list-of-links--li-height: 54px;
+	--wporg--style--list-of-links--line-height: 1.92;
+	--wporg--style--list-of-links--border-opacity: 0.2;
+	padding: var(--wporg--style--list-of-links--offset);
+	list-style: none;
+	font-size: var(--wp--preset--font-size--extra-large);
+	line-height: var(--wporg--style--list-of-links--line-height);
+	background-color: var(--wp--preset--color--blueberry-1);
```

Comment:
Do we need to specify background-color for users? I saw that in https://github.com/WordPress/wporg-parent-2021/pull/28/files, we didn't do so.